### PR TITLE
chore: update bridge and relayer to latest for e2e testing

### LINKED ISSUE #794: [TASK] Remove deprecated relayers and add Webb dedicated relayers to relayer selection

### PR DESCRIPTION
**Pull request by dutterbutter**
## Summary of changes
- Updates anchor addresses to latest deployments 
- Updates relayer endpoints to latest deployments 

@AhmedKorim can you confirm that these are the only two places that need to be updated.

### Reference issue to close (if applicable)
- Closes #794 

-----
### Code Checklist 

- [ ] Tested
- [ ] Documented

### CODE REVIEW

File: libs/dapp-config/src/anchors/anchor-config.ts [hunk 1]
```diff
@@ -40,22 +38,19 @@ export const anchorsConfig: ApiConfig['anchors'] = {
 
 export const anchorDeploymentBlock: Record<number, Record<string, number>> = {
   [calculateTypedChainId(ChainType.EVM, EVMChainId.Goerli)]: {
-    '0xdc9f140987073619d64898be00143dd312cdc71a': 7865480,
+    '0xbd7e08ef54dfb43e9a86eb12a3ecab7564dee931': 8048194,
   },
   [calculateTypedChainId(ChainType.EVM, EVMChainId.Sepolia)]: {
-    '0xa5672fdfe6700452a828c514a9d5b3b4dc5230bf': 2196862,
+    '0x965c94a6e1b713e751164d2ba09aa0306f48ee74': 2386639,
```
Comment (salman01zp) [inline libs/dapp-config/src/anchors/anchor-config.ts:44]:
```suggestion
    '0x965c94a6e1b713e751164d2ba09aa0306f48ee74': 8048141,
```

**Review by salman01zp — COMMENTED** [review]
LG.. Just need to update sepolia block number deployed at.

**Comment by dutterbutter** [comment]
Withdraw crashes with new deployments it seems see #817

**Comment by dutterbutter** [comment]
Closing in favour of #806

**Pull request closed without merging**